calc/src/mechanics/gen78.ts

```diff
@@ -361,10 +371,27 @@ export function calculateSMSS(
     basePower = r >= 4 ? 150 : r >= 3 ? 120 : r >= 2 ? 80 : r >= 1 ? 60 : 40;
     desc.moveBP = basePower;
     break;
+  case 'Expanding Force':
+    basePower = move.bp * (field.hasTerrain("Psychic") ? 2 : 1);
+    move.target = field.hasTerrain("Psychic") ? "allAdjacentFoes" : "normal";
+    desc.moveBP = basePower;
+    break;
```

Comment:
This only applies when the user is grounded (aka not holding air balloon/doesnt have levitate/not flying type), idk if there's a way to check that

this is the only move it's confirmed to apply to for now

Reply:
we have `isGrounded`